pkg/grpc/label_store/client/client.go

```diff
@@ -0,0 +1,126 @@
+/*
+This package provides an client that implements the normal labels applicator
+functions but maps them onto a grpc server call
+*/
+package client
+
+import (
+	"crypto/tls"
+	"time"
+
+	label_protos "github.com/square/p2/pkg/grpc/label_store/protos"
+	"github.com/square/p2/pkg/labels"
+	"github.com/square/p2/pkg/logging"
+
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/credentials"
+	klabels "k8s.io/kubernetes/pkg/labels"
+)
+
+type Client struct {
+	conn             *grpc.ClientConn
+	labelStoreClient label_protos.LabelStoreClient
+	logger           logging.Logger
+}
+
+func NewClient(grpcAddress string, tlsConfig *tls.Config, logger logging.Logger) (Client, error) {
```

Comment:
I think I should change this to take the `conn` as the argument to support things like `grpc.WithBalancer()`